core/src/main/java/org/elasticsearch/action/termvectors/TermVectorsRequest.java

```diff
@@ -60,6 +61,22 @@
  */
 public class TermVectorsRequest extends SingleShardRequest<TermVectorsRequest> implements RealtimeRequest {
 
+    private static final ParseField INDEX = new ParseField("_index");
+    private static final ParseField TYPE = new ParseField("_type");
+    private static final ParseField ID = new ParseField("_id");
+    private static final ParseField ROUTING = new ParseField("routing", "_routing");
+    private static final ParseField PARENT = new ParseField("parent", "_parent");
+    private static final ParseField VERSION = new ParseField("version", "_version");
+    private static final ParseField VERSION_TYPE = new ParseField("version_type", "_version_type", "_versionType", "versionType");
```

Comment:
We've dropped support for camel case most other places a while ago. I can see that we kept it here. This change will start to emit a deprecation warning which is great. I'm fine with keeping this camel case for another major version and dropping it then. It might be good to add a test that tries to use the deprecated camel case that fails when the current version's major is `> 7`. Then when we bump the version we'll drop the camel case.